### PR TITLE
[FIX] update plot_surface_101 example

### PR DESCRIPTION
**Pull request by Remi-Gau**
- Closes none

Changes proposed in this pull request:

- plotting can be done by passing directly a surface image: mesh is optional in that case
- styling to force "narrower" code (views better on smaller screens)

### CODE REVIEW

**Comment by Remi-Gau** [comment]
[HTML render of the updated example](https://output.circle-artifacts.com/output/job/51f53bce-f2ea-4d14-942f-986471a0b73c/artifacts/0/dev/auto_examples/00_tutorials/plot_surface_101.html)

**Comment by emdupre** [comment]
On mobile it looks as though plotting is only done for the first surface image (attaching screenshots). Is this expected ?

image

image

**Comment by Remi-Gau** [comment]
> Is this expected ?

Same on laptop.
I don't think it is expected: let me add another show to see if that helps

**Comment by man-shu** [comment]
I think `view_surf` directly shows the plots without the `show()`

**Comment by man-shu** [comment]
Just confirmed that locally:
Screenshot 2024-12-02 at 10 10 13 AM

File: examples/00_tutorials/plot_surface_101.py [hunk 1]
```diff
@@ -125,25 +139,29 @@
...
 # Plot the left part
-plotting.view_surf(
-    surf_mesh=surface_image.mesh, surf_map=surface_image, hemi="left"
+view_surf(
+    surf_map=surface_image,
+    hemi="left",
 )
 
+show()
+
```
Comment (man-shu) [inline examples/00_tutorials/plot_surface_101.py:156]:
```suggestion
```

Comment (man-shu) [reply examples/00_tutorials/plot_surface_101.py:156]: So we can just remove them

**Review by man-shu — APPROVED** [review]
Otherwise LGTM!

**Comment by Remi-Gau** [comment]
all the plots are now showing: merging

**Pull request merged**